cmd/status/security/security.go

```diff
@@ -0,0 +1,203 @@
+package status_security
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/rs/zerolog"
+	"github.com/spf13/cobra"
+
+	"github.com/quickfixgo/enum"
+	"github.com/quickfixgo/field"
+	"github.com/quickfixgo/fixt11"
+	"github.com/quickfixgo/quickfix"
+
+	"sylr.dev/fix/config"
+	"sylr.dev/fix/pkg/cli/complete"
+	"sylr.dev/fix/pkg/dict"
+	"sylr.dev/fix/pkg/errors"
+	"sylr.dev/fix/pkg/initiator"
+	"sylr.dev/fix/pkg/initiator/application"
+	"sylr.dev/fix/pkg/utils"
+)
+
+var (
+	optionSecurityStatReqID string
+	optionSubType           string
+	optionSymbol            string
+)
+
+var StatusSecurityCmd = &cobra.Command{
+	Use:               "security",
+	Short:             "security status",
+	Long:              "Send a security Session Status Request after initiating a session with a FIX acceptor.",
+	Args:              cobra.ExactArgs(0),
+	ValidArgsFunction: cobra.NoFileCompletions,
+	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+		err := Validate(cmd, args)
+		if err != nil {
+			return err
+		}
+
+		if cmd.HasParent() {
+			parent := cmd.Parent()
+			if parent.PersistentPreRunE != nil {
+				err = parent.PersistentPreRunE(parent, args)
+				if err != nil {
+					return err
+				}
+			}
+		}
+
+		return nil
+	},
+	RunE: Execute,
+}
+
+func init() {
+	StatusSecurityCmd.Flags().StringVar(&optionSymbol, "symbol", "MONA_EUR", "Symbol")
```

Comment:
Do not set a default value, this should be a mandatory option.
```suggestion
	StatusSecurityCmd.Flags().StringVar(&optionSymbol, "symbol", "", "Symbol")
```